EIPS/rentcollector.md

```diff
@@ -0,0 +1,73 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Rent Collection for Blockchain Data
+    Author: Daniel A. Nagy <daniel@ethereum.org>, Zsolt Felfoldi <zsolt@ethereum.org>
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-11-01
+
+
+## Simple Summary
+Introducing a way to collect rent from accounts that is proportional 
+to the cost that they impose on the network by storing a large state
+over a long period of time.
+
+## Abstract
+A new global parameter called `rent` similar to `gasLimit` that can be 
+modified by miner vote is introduced. It is initialized at 0 and is 
+assumed to have been zero before its introduction. The miner validating 
+a particular block can increase rent in a bounded way at the cost of 
+forfeiting a proportional part of the mining reward. Similarly, if 
```

Comment:
Forfeiting the reward for the present block, or for all subsequent blocks?

Reply:
The idea was for only the present block, because doing it for all subsequent blocks would mean that miners would have no incentive to increase the rent at all, as the sum of the increased rent and decreased mining reward would get smaller (if the proportion is smaller than 1, as it should be).
Should I make it explicit?